roles/debian-common/files/ansible.cfg

```diff
@@ -0,0 +1,167 @@
+# config file for ansible -- http://ansible.com/
```

Comment:
I really don't think we should duplicate this file for every OS. We should have a common rule that runs on all OSes. That being said, since this was already the case, it's not particularly a problem with this change.